infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutSwitchController.kt

```diff
@@ -97,6 +97,18 @@ class LayoutSwitchController(
         return publicationService.validateSwitches(ids, publishType)
     }
 
+    @PreAuthorize(AUTH_UI_READ)
+    @GetMapping("/{publishType}/validation/{bbox}")
+    fun validateSwitchesInTile(
+        @PathVariable("publishType") publishType: PublishType,
+        @PathVariable("bbox") bbox: BoundingBox,
+    ): List<ValidatedAsset<TrackLayoutSwitch>> {
+        logger.apiCall("validateSwitchesInTile", "publishType" to publishType, "bbox" to bbox)
+        val filter = switchFilter(null, null, null, bbox, false)
```

Comment:
Hmm, olipas jotenkin odottamaton funkkari jolle passaillaan nullia/falsea moneen kohtaan. Tässä voisi varmaankin nimetä parametrin ja jättää muut pois, kun `switchFilter` asettikin jo defaultteja? (Nyt lukija siis joutuu/joutui tutkimaan mitä nullit/falset merkkailevat) 

```
val filter = switchFilter(bbox = bbox)
```

Vastaavasti `filter`-niminen muuttuja on vähän niin ja näin, vaikka onkin käytössä välittömästi seuraavalla rivillä ja ainoastaan siinä. Vähän vielä jatkotulkintahelpoitukseksi tuon voisi varmaankin nakata vielä suoraan tuohon filtterikutsuun jolloin muuttujaakaan ei enää tarvittaisi:
```
val switches = switchService
    .listWithStructure(publishType, false)
    .filter(switchFilter(bbox = bbox))

Reply:
Mmh, totta. Voisihan tuota tosiaan vielä kaunistellakin. Tainnut jäädä vähän TypeScript-vaihde silmään kun unohtuu että funktioille voi antaa parametrit nimettyinäkin.